subz_commands_aclr8.py

```diff
@@ -30,6 +34,41 @@ def remove_temp(self, path, is_temporary):
     if is_temporary:
       os.remove(path)
 
+class SubzAclr8TestsCoupled(sublime_plugin.WindowCommand):
```

Comment:
there are a few nitpicks, but maybe they would be best proposed together:
```python
class SubzAclr8TestsCoupled(sublime_plugin.WindowCommand):
  def run(self):
    view = sublime.active_window().active_view()

    if not ion_is_saved(view):
      sublime.message_dialog("Please save all changes before running tests")
      return

    file_path = view.file_name()
    working_dir, _fname = os.path.split(file_path)

    with ExitStack() as stack:
      if "coupled" not in working_dir:
        tmp_dir_root = stack.enter_context(TemporaryDirectory())
        tmp_dir = os.path.join(tmp_dir_root, "coupled")
        copy_tree(working_dir, tmp_dir)
        working_path = tmp_dir

      run_aclr8_command("test " + working_path + "/", on_success=lambda output: append_output_panel(output + "All tests passed."))


```
Using the `with` statement  (for scoped resources) allows removing the callbacks. What do you think?

(required imports
```
from contextlib import ExitStack
from tempfile import TemporaryDirectory
```
)